### PR TITLE
Faster impact tsunami on roads

### PR DESCRIPTION
**Pull request by timlinux**

### CODE REVIEW

File: safe/impact_functions/inundation/tsunami_raster_road/impact_function.py [hunk 1]
```diff
@@ -95,8 +99,12 @@ def _raster_to_vector_cells(raster, ranges, output_crs):
 
     # prepare coordinate transform to reprojection
     ct = QgsCoordinateTransform(raster.crs(), output_crs)
-
     rd = 0
+    # Hack because QGIS does not return the
+    # nodata value from the dataset properly in the python API
+    dataset = gdal.Open(raster.source())
+    no_data = dataset.GetRasterBand(1).GetNoDataValue()
+    dataset = None  # close the dataset
```
Comment (Gustry) [inline safe/impact_functions/inundation/tsunami_raster_road/impact_function.py]: Why not using `del dataset` ?

File: safe/impact_functions/inundation/tsunami_raster_road/impact_function.py [hunk 2]
```diff
@@ -154,6 +167,9 @@ def _raster_to_vector_cells(raster, ranges, output_crs):
                 features = []
     # Add the latest features
     vl.dataProvider().addFeatures(features)
+    # For debugging
+    # from qgis.core import QgsMapLayerRegistry
+    # QgsMapLayerRegistry.instance().addMapLayer(vl)
```
Comment (Gustry) [inline safe/impact_functions/inundation/tsunami_raster_road/impact_function.py]: These lines can be rewritten easily. If we let these kind of lines, we will find a lot in a few months/years.

File: safe/impact_functions/inundation/tsunami_raster_road/impact_function.py (hunk 1 above)
Comment (timlinux) [inline safe/impact_functions/inundation/tsunami_raster_road/impact_function.py]: I followed the goal coding guide for this

File: safe/impact_functions/inundation/tsunami_raster_road/impact_function.py (hunk 2 above)
Comment (timlinux) [inline safe/impact_functions/inundation/tsunami_raster_road/impact_function.py]: Agreed - will will delete it

File: safe/impact_functions/inundation/tsunami_raster_road/impact_function.py (hunk 1 above)
Comment (Gustry) [inline safe/impact_functions/inundation/tsunami_raster_road/impact_function.py]: Ok. It's not very pythonic :)
I just read http://gis.stackexchange.com/questions/80366/why-close-a-dataset-in-gdal-python but it's not the coding guide

**Pull request merged**